mods/default/locale/default.de.tr

```diff
@@ -0,0 +1,211 @@
+# textdomain: default
+Locked Chest=Abgeschlossene Truhe
+Locked Chest (owned by @1)=Abgeschlossene Truhe (Eigentum von @1)
+You do not own this chest.=Ihnen gehört diese Truhe nicht.
+a locked chest=eine abgeschlossene Truhe
+Chest=Truhe
+Stick=Stock
+Paper=Papier
+"@1" by @2=„@1“ von @2
+Book=Buch
+Book with Text=Buch mit Text
+Skeleton Key=Skelettschlüssel
+Key to @1's @2=Schlüssel für @2 von @1
+Coal Lump=Kohleklumpen
+Iron Lump=Eisenklumpen
+Copper Lump=Kupferklumpen
+Tin Lump=Zinnklumpen
+Mese Crystal=Mesekristall
+Gold Lump=Goldklumpen
+Diamond=Diamant
+Clay Lump=Tonklumpen
+Steel Ingot=Stahlbarren
+Copper Ingot=Kupferbarren
+Tin Ingot=Zinnbarren
+Bronze Ingot=Bronzebarren
+Gold Ingot=Goldbarren
+Mese Crystal Fragment=Mesekristallfragment
+Clay Brick=Tonziegel
+Obsidian Shard=Obsidianscherbe
+Flint=Feuerstein
+Blueberries=Blaubeeren
+Furnace is empty=Ofen ist leer
+100% (output full)=100% (Ausgabe voll)
+@1%=@1%
+Empty=Leer
+Not cookable=Nicht kochbar
+Furnace active=Ofen aktiv
+Furnace inactive=Ofen inaktiv
+(Item: @1; Fuel: @2)=(Gegenstand: @1; Brennstoff: @2)
+Furnace=Ofen
+Stone=Stein
+Cobblestone=Kopfsteinpflaster
+Stone Brick=Steinziegel
+Stone Block=Steinblock
+Mossy Cobblestone=Mosiges Kopfsteinpflaster
+Desert Stone=Wüstenstein
+Desert Cobblestone=Wüstenkopfsteinpflaster
+Desert Stone Brick=Wüstensteinziegel
+Desert Stone Block=Wüstensteinblock
+Sandstone=Sandstein
+Sandstone Brick=Sandsteinziegel
+Sandstone Block=Sandsteinblock
+Desert Sandstone=Wüstensandstein
+Desert Sandstone Brick=Wüstensandsteinziegel
+Desert Sandstone Block=Wüstensandsteinblock
+Silver Sandstone=Silbersandstein
+Silver Sandstone Brick=Silbersandsteinziegel
+Silver Sandstone Block=Silbersandsteinblock
+Obsidian=Obsidian
+Obsidian Brick=Obsidianziegel
+Obsidian Block=Obsidianblock
+Dirt=Erde
+Dirt with Grass=Erde mit Gras
+Dirt with Grass and Footsteps=Erde mit Gras und Fußstapfen
+Dirt with Dry Grass=Erde mit trockenem Gras
+Dirt with Snow=Erde mit Schnee
+Dirt with Rainforest Litter=Erde mit Regenwaldboden
+Dirt with Coniferous Litter=Erde mit Nadelwaldboden
+Dry Dirt=Trockene Erde
+Dry Dirt with Dry Grass=Trockene Erde mit trockenem Gras
+Permafrost=Permafrost
+Permafrost with Stones=Permafrost mit Steinen
+Permafrost with Moss=Permafrost mit Moos
+Sand=Sand
+Desert Sand=Wüstensand
+Silver Sand=Silbersand
+Gravel=Kies
+Clay=Ton
```

Comment:
Lehm/Tonerde? To me those would be clearer, with less risk of misinterpretations.